GestureHandler.js

```diff
@@ -352,7 +322,7 @@ function createHandler(
       }
 
       const child = React.Children.only(this.props.children);
-      let children = child.props.children;
+      let children = GestureHandlerModule.getChildren(this.props);
```

Comment:
This line seem to be changing behavior on mobile platforms. It returns a single child on mobile as far as I can see. Have you tested this on mobile with overlay debugger ON?